### PR TITLE
Fix landing page with invalid timeline

### PR DESCRIPTION
**Pull request by n0izn0iz**
https://bluenove.atlassian.net/browse/DEVAS-934

### CODE REVIEW

File: assembl/static2/js/app/main.jsx [hunk 1]
```diff
@@ -27,13 +30,10 @@ class Main extends React.Component {
       const currentPhase = debateData.timeline.filter((phase) => {
         return phase.identifier === currentPhaseIdentifier;
       });
-      isRedirectionToV1 = currentPhase[0].interface_v1;
+      isRedirectionToV1 = currentPhase[0] && currentPhase[0].interface_v1;
     }
     this.props.addRedirectionToV1(isRedirectionToV1);
   }
-  componentDidMount() {
-    window.addEventListener('scroll', this.displayHeader);
```
Comment (ayyazdaniaryan) [inline assembl/static2/js/app/main.jsx]: This isn't relevant any longer?

Comment (n0izn0iz) [reply assembl/static2/js/app/main.jsx]: this.displayHeader is undefined I can't see where it could be.

Comment (n0izn0iz) [reply assembl/static2/js/app/main.jsx]: there is no "displayHeader" string in the project code except here

Comment (ayyazdaniaryan) [reply assembl/static2/js/app/main.jsx]: Okay, dead code. Good kill.

**Pull request merged**